### PR TITLE
Phishing classifier Ui improvements

### PR DESCRIPTION
**Pull request by lperryp**
Standardizing the terminology of "Confidence threshold" 
Fixing error messages 
fixes https://github.com/demisto/etc/issues/27011
## XSOAR [Partner](https://xsoar.pan.dev/docs/partners/become-a-tech-partner)?
- [ ] The title must be in the following format: **[YOUR_PARTNER_ID] short description**.

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py [hunk 1]
```diff
@@ -90,7 +90,7 @@ def predict_phishing_words(model_name, model_store_type, email_subject, email_bo
     explain_result_hr = dict()
     explain_result_hr['TextTokensHighlighted'] = highlighted_text_markdown
     explain_result_hr['Label'] = predicted_label
-    explain_result_hr['Probability'] = "%.2f" % predicted_prob
+    explain_result_hr['Confidence'] = "%.2f" % predicted_prob
```
Comment (erezh31) [inline Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:93]: please add another field called confidence and don't replace the current one so we won't have backward compatabilites issues. Also you need to update the outputs - add new output (also have this, the old one)

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py [hunk 2]
```diff
@@ -122,12 +122,16 @@ def find_words_contain_tokens(positive_tokens, words_to_token_maps):
...
+    confidence_threshold = float(demisto.args().get("labelProbabilityThreshold", confidence_threshold))
+    confidence_threshold = float(demisto.args().get("confidenceThreshold", confidence_threshold))
+
     result = predict_phishing_words(demisto.args()['modelName'],
                                     demisto.args()['modelStoreType'],
                                     demisto.args().get('emailSubject', ''),
                                     demisto.args().get('emailBody', '') or demisto.args().get('emailBodyHTML', ''),
                                     int(demisto.args()['minTextLength']),
-                                    float(demisto.args().get("labelProbabilityThreshold", 0)),
+                                    confidence_threshold,
```
Comment (erezh31) [inline Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.py:134]: you need to support the deprecated argument also

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml [hunk 3]
```diff
@@ -65,11 +65,18 @@ args:
   secret: false
 - default: false
   defaultValue: '0.8'
-  description: The label probability threshold.
+  description: The label probability threshold. Deprecated, please use quivalnet "confidenceThreshold" arguement.
```
Comment (erezh31) [inline Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml:68]: you need to mark it as deprecated: true, look on other samples

File: Packs/ML/Scripts/DBotPredictOutOfTheBox/DBotPredictOutOfTheBox.yml [hunk 4]
```diff
@@ -42,11 +42,18 @@ args:
   secret: false
 - default: false
   defaultValue: '0'
-  description: The label probability threshold.
+  description: The label probability threshold. Deprecated, please use quivalnet "confidenceThreshold" arguement.
```
Comment (erezh31) [inline Packs/ML/Scripts/DBotPredictOutOfTheBox/DBotPredictOutOfTheBox.yml:45]: same

**Review by erezh31 — CHANGES_REQUESTED** [review]

**Comment by lperryp** [comment]
Changing DBotPRedictPhishingWords to run once as requested at: https://github.com/demisto/etc/issues/27011

**Review by erezh31 — APPROVED** [review]

**Comment by kirbles19** [comment]
@lperryp Done

File: Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml [hunk 5]
```diff
@@ -126,5 +133,7 @@ tags:
 timeout: '0'
 type: python
 dockerimage: demisto/dl:1.3
+runas: DBotWeakRole
```
Comment (meirwah) [inline Packs/Base/Scripts/DBotPredictPhishingWords/DBotPredictPhishingWords.yml:136]: why you need to add this?

**Comment by Itay4** [comment]
build fails on non relevant docker image validation
merging

**Pull request merged**